src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -4565,20 +4598,24 @@ public static void Main(string[] args)
     private static object M() => null;
 }";
             CreateCompilationWithMscorlib45(program).VerifyDiagnostics(
+                // (9,18): error CS0150: A constant value is expected
+                //             case true ? x3 : 4:
+                Diagnostic(ErrorCode.ERR_ConstantExpected, "true ? x3 : 4").WithLocation(9, 18)
                 );
         }
 
-        [Fact(Skip = "This test generator is capable of producing crashes that are not yet fixed; see, for example, Fuzz5815")]
+        [Fact(Skip = "This test generator runs too long to use frequently")]
```

Comment:
As long as we are changing this test please file a bug and use it in the skip reason.  If this is useful we should work on the perf here until we can get it to be a part of a normal run.  Same work was done for some @agocke generated tests.